authors/apps/articles/tests/tests_articles.py

```diff
@@ -0,0 +1,128 @@
+
+from django.urls import reverse, path, include
+from django.utils import timezone
+from rest_framework import status
+from rest_framework.test import APITestCase, URLPatternsTestCase
+from rest_framework.test import APIClient
+from authors.apps.articles.models import Article
+from authors.apps.authentication.models import User
+from django.test import TestCase
+
+
+class ArticleTestCase(APITestCase):
+    """Test for articley views."""
+
+    def setUp(self):
+        """Initialize setup."""
```

Comment:
@KabohaJeanMark Lets use fixtures here sync up with @mwinel and @3Nakajugo. Left a similar comment on their pr's.